vendor/init.bat

```diff
@@ -312,18 +316,27 @@ if not exist "%initialConfig%" (
     echo Creating user startup file: "%initialConfig%"
     (
 echo :: use this file to run your own startup commands
-echo :: use  in front of the command to prevent printing the command
+echo :: use in front of the command to prevent printing the command
+echo.
+echo :: the next two lines is for "%%flag_exists%%" shortcut, a custom arguments handler
+echo :: don't remove it if you need it
```

Comment:
Since `init.bat` should be left unchanged by the users, I doubt they'll edit this file. In any case, we should discourage users from directly editing this file.

Reply:
Yeah, but I think it has nothing to do with this PR.
In my point of view, this PR just **isn't** a BREAK CHANGE.
They can find this new feature in CHANGELOG or README.md.

We just recommend them not to edit `init.bat`, if they modified `init.bat`, it's their responsibility to check carefully while upgrading.

About *we should discourage users from directly editing this file.*. I don't think it has something to do with this PR.